packages/features/ee/organizations/pages/settings/appearance.tsx

```diff
@@ -79,134 +75,79 @@ const OrgAppearanceView = ({
   };
 
   return (
-    <LicenseRequired>
-      <Meta
-        title={t("appearance")}
-        description={t("appearance_org_description")}
-        borderInShellHeader={false}
-      />
```

Comment:
Why did you remove LicenseRequired and Meta ?

Reply:
and only

Reply:
We used the component in the profile section, where LicenseRequired and Meta were already used as parents.